helper/report.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/**
+ * Class helper_plugin_loglog_report
+ */
+class helper_plugin_loglog_report extends \dokuwiki\Extension\Plugin
+{
+    /**
+     * @var \helper_plugin_loglog_main
+     */
+    protected $mainHelper;
+
+    /**
+     * @var \helper_plugin_loglog_logging
+     */
+    protected $logHelper;
+
+    public function __construct()
+    {
+        $this->mainHelper = $this->loadHelper('loglog_main');
+        $this->logHelper = $this->loadHelper('loglog_logging');
+    }
+
+    public function create()
```

Comment:
missing doc block. This also does not only create the report it also sends it.

Reply:
Fixed